{"name": "test_divisible_by_6", "status": "failed", "statusDetails": {"message": "AssertionError: assert (39 % 12) == 1", "trace": "input_value = 39\n\n    def test_divisible_by_6(input_value):\n>       assert input_value % 12 == 1\nE       assert (39 % 12) == 1\n\ntest_div_by_3_6.py:8: AssertionError"}, "start": 1678268172514, "stop": 1678268172514, "uuid": "0d4c1102-dc10-4e91-97fe-bbd5a2b54dbe", "historyId": "32b8478b746a25bb1aabbecba9086198", "testCaseId": "a8104418aab4ba63e6decb0cab16bf8e", "fullName": "test_div_by_3_6#test_divisible_by_6", "labels": [{"name": "suite", "value": "test_div_by_3_6"}, {"name": "host", "value": "BAN-LAP-VREDDY"}, {"name": "thread", "value": "7936-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_div_by_3_6"}]}